apps/prairielearn/src/sync/fromDisk/authors.sql

```diff
@@ -0,0 +1,14 @@
+-- BLOCK insert_authors
+INSERT INTO
+  authors (author_string)
+SELECT
+  (a ->> 0)::text
+FROM
+  UNNEST($authors::jsonb[]) AS a
+ON CONFLICT (author_string) DO NOTHING;
+
+-- BLOCK select_authors
+SELECT
+  *
+FROM
+  authors;
```

Comment:
This is obviously terrible, we only want to select the authors that are listed on questions in the current course. Is there an easy way to do something like `WHERE author.author_string IN <some jsonb array>` ? 

I always get tripped up on the syntax when passing in JSONB.